src/App/Containers/Modals/index.js

```diff
@@ -0,0 +1,28 @@
+import React       from 'react';
+import { connect } from 'Stores/connect';
+
+const DenialOfServiceModal = React.lazy(() => import(/* webpackChunkName: "DenialOfServiceModal" */'../DenialOfServiceModal'));
+const MarketUnavailableModal = React.lazy(() => import(/* webpackChunkName: "MarketUnavailableModal" */'../MarketUnavailableModal'));
+const ServicesErrorModal = React.lazy(() => import(/* webpackChunkName: "ServicesErrorModal" */'../ServicesErrorModal'));
+const UnsupportedContractModal = React.lazy(() => import(/* webpackChunkName: "UnsupportedContractModal" */'../UnsupportedContractModal'));
+
+const Modals = ({
+    is_denial_of_service_modal_visible,
+    is_unsupported_contract_modal_visible,
+    is_market_unavailable_visible,
+    is_services_error_visible,
+}) => (
+    <React.Fragment>
+        {is_unsupported_contract_modal_visible && <UnsupportedContractModal /> }
```

Comment:
@easteregg Shouldn't they be wrapped by `React.Suspense`?

Reply:
AFAIK, `React.Suspense` is deep, so if it existed in some upper scope, no need to call it for every single one.